### ISSUE TITLE
Subtitles

### ISSUE THREAD

**Issue by ThomasCSR** [RandomEngy/VidCoder#324]
It's possible to add a subtitle .srt file to a session but it's not possible to use subtitles within a MKV file.
That's odd
By the way: the "report a bug in the Help menu doesn't work.

**Comment by RandomEngy** [comment]
Are you saying you can't burn .srt to the picture when outputting MKV?

Also what version are you using? Portable, installer, store? Beta or non-beta? "Report a bug" seems to work for me.

**Comment by ThomasCSR** [comment]
No. I have some mkv-videos with internal subtitles (in fact the are .srt).
I want to convert them to H264 witch works fine. But the subtitles are
missing in the converted videos. I seems VidCoder ignores them).
I have to extract them from the source videos and manually mux them to the
converted videos.
What I want is the option to bring subtitles from the source to the output.

Venlig hilsen / best regards
Thomas Richardt

Den 7. mar. 2018 16.44 skrev "David Rickard" <notifications@github.com>:

Are you saying you can't burn .srt to the picture when outputting MKV?

Also what version are you using? Portable, installer, store? Beta or
non-beta? "Report a bug" seems to work for me.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/RandomEngy/VidCoder/issues/324#issuecomment-371180852>,
or mute the thread

**Comment by ThomasCSR** [comment]
Report a bug leds to a nonexisting site.

Venlig hilsen / best regards
Thomas Richardt

Den 7. mar. 2018 16.44 skrev "David Rickard" <notifications@github.com>:

**Comment by RandomEngy** [comment]
What URL does it send you to?

**Comment by Ghastor** [comment]
It's possible ti select subtitles in main Windows with flags, instead to open and the close window? All subtitles visible and selectables on file open

**Comment by RandomEngy** [comment]
That's actually something I've already been planning to do. I want to make the subtitles/audio/chapters collapsible sections that can show fully as soon as the source is open.

**Comment by Ghastor** [comment]
Thanks !!

**Comment by RandomEngy** [comment]
The ability to edit subtitles without opening another dialog is part of the recent 4.6 Beta.

As for source subtitles missing from the detected subtitles list, can you check the behavior with HandBrake UI? I rely on their library for subtitle detection.

**Comment by RandomEngy** [comment]
That's a decent idea but I like having a consistent UI scheme for showing the different parts of the source video. There is "unused space" but only when the details are opened. Though if you want to go ahead with a mockup or pull request, go ahead.
